downstream/modules/security/proc-creating-ids-rule.adoc

```diff
@@ -0,0 +1,98 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: proc-doing-procedure-a.adoc
+* ID: [id="doing-procedure-a_{context}"]
+* Title: = Doing procedure A
+
+The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+////
+
+[id="proc-creating-ids-rule_{context}"]
+
+= Creating a new IDS rule
+
+Use the `ids_rule` role to manage your rules and signatures for IDS. For example, you can set a new rule that looks for a certain pattern aligning with a previous attack on your firewall.
+
+[NOTE]
+====
+Currently, the `ids_rule` role only supports Snort IDS.
+====
+
+.Prerequisites
+
+* You need `root` privileges to make any changes on the Snort server.
+
+.Procedure
+
+. Install the `ids_rule` role using the ansible-galaxy command:
++
+----
+$ ansible-galaxy install ansible_security.ids_rule
+----
+
+. Create a new playbook file titled `add_snort_rule.yml`. Set the following parameters:
++
+----
+- name: Add Snort rule
+  hosts: snort
+----
+
```

Comment:
Would it make sense to group the code before `tasks` into one example code block?